portfolio/src/main/java/com/google/sps/servlets/HomeServlet.java

```diff
@@ -45,13 +44,18 @@ public void doGet(HttpServletRequest request, HttpServletResponse response) thro
        
       response.getWriter().println("<p>Logout <a href=\"" + logoutUrl + "\">here</a>.</p>");
     } else {
-      String urlToRedirectToAfterUserLogsIn = "/";
-      String loginUrl = userService.createLoginURL(urlToRedirectToAfterUserLogsIn);
-
-      response.getWriter().println("<p>Hello stranger.</p>");
-      
-     
-      response.getWriter().println("<p>Login <a href=\"" + loginUrl + "\">here</a>.</p>");
-    }
+       String urlToRedirectToAfterUserLogsIn = "/";
+       String loginUrl = userService.createLoginURL(urlToRedirectToAfterUserLogsIn);
+       response.getWriter().println("<p>Hello stranger.</p>");
+       response.getWriter().println("<p>Before make comment, Please login first!</p>");
+       response.getWriter().println("<p>Login <a href=\"" + loginUrl + "\">here</a>.</p>");
+      //response.sendRedirect("/"+loginUrl);
+     // RequestDispatcher dispatcher = getServletContext().getRequestDispatcher("/"+loginUrl);
+    //dispatcher.forward(request, response);
+    //try{request.getRequestDispatcher("/"+loginUrl).forward(request, response);}catch(Exception e) {
+						
+	//e.printStackTrace();}
   }
+  
+}
```

Comment:
Is something off with your indentation? Two closing curly brackets at the same level of indentation here seems strange to me.